js/renovation/ui/common/__tests__/widget.test.tsx

```diff
@@ -544,6 +546,48 @@ describe('Widget', () => {
         });
       });
 
+      describe('windowResizeEffect', () => {
+        afterEach(() => {
+          jest.clearAllMocks();
+        });
+
+        it('should subscribe on window.onresize event', () => {
+          const onDimensionChanged = jest.fn();
+          const widget = new Widget({ onDimensionChanged });
+
+          const dispose = widget.windowResizeEffect() as DisposeEffectReturn;
+
+          expect(resizeCallbacks.add).toBeCalledTimes(1);
+          expect(resizeCallbacks.remove).toBeCalledTimes(0);
+
+          dispose?.();
+        });
+
+        it('should return window.onresize unsubscribe callback', () => {
+          const onDimensionChanged = jest.fn();
+          const widget = new Widget({ onDimensionChanged });
+
+          const dispose = widget.windowResizeEffect() as DisposeEffectReturn;
+
+          dispose?.();
+
+          expect(resizeCallbacks.remove).toBeCalledTimes(1);
+
+          const callbackPassedToAdd = (resizeCallbacks as any).add.mock.calls[0][0];
```

Comment:
expect(resizeCallbacks).toHaveBeenCalledWith(resizeCallback);

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/16330/commits/30ed274aeec98ed88b062d88c0e35fb7b77451f7